### PR TITLE
Additional topics

### PR DESCRIPTION
**Pull request by biancabnd**
---
 
## Basic Info
 
| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | (Closes #IssueNumber) |
| Primary OS tested on | (Ubuntu, MacOS, Windows) |
| Drone platform tested on | DJI M300 RTK  |
 
---
 
## Description of contribution in a few bullet points
 
* Added obstacle avoidance topic
* Added altitude related topics (altitude sea level, altitude barometric and  altitude of home point)
* Added a retry strategy to the initialization function which sometimes returns a timeout error

## How Has This Been Tested?

* Tested with DJI M300 drone

## Description of documentation updates required from your changes
 
* Documentation has been updated accordingly

---
 
## Screenshots (if appropriate):

### CODE REVIEW

**Review by amoramar — COMMENTED** [review]
Great to see more capabilities added to the PSDK wrapper! 🦾 🔥 
I addressed very, very minor details about the comments on the Doxygen documentation for the newly added callback methods. I can push those changes if you are okay with that.

_**P.S.** I have not had the chance to test this in HITL, I can do that as soon as I get access to the hardware._

**Comment by biancabnd** [comment]
> Great to see more capabilities added to the PSDK wrapper! 🦾 🔥 I addressed very, very minor details about the comments on the Doxygen documentation for the newly added callback methods. I can push those changes if you are okay with that.
> 
> _**P.S.** I have not had the chance to test this in HITL, I can do that as soon as I get access to the hardware._

Thanks for the review @amoramar ! And sure, you can push the changes! Many thanks

File: psdk_wrapper/include/psdk_wrapper/psdk_wrapper.hpp [hunk 1]
```diff
@@ -965,6 +977,58 @@ class PSDKWrapper : public rclcpp_lifecycle::LifecycleNode
   T_DjiReturnCode acceleration_body_raw_callback(
       const uint8_t* data, uint16_t data_size,
       const T_DjiDataTimestamp* timestamp);
+  /**
+   * @brief Retrieves the obstacle avoidance data around the vehicle at a
+   * frequency up to 100 Hz. It also provides a health flag for each sensor
+   * direction. Please refer to the msg
+   * definition sensor_msgs::msg::RelativeObstacleInfo for more details.
```
[the remark this answers is not in the document]
Comment (biancabnd) [reply psdk_wrapper/include/psdk_wrapper/psdk_wrapper.hpp:984]: you are completely right!

File: psdk_wrapper/include/psdk_wrapper/psdk_wrapper.hpp [hunk 2]
```diff
@@ -965,6 +977,58 @@ class PSDKWrapper : public rclcpp_lifecycle::LifecycleNode
...
+   * @param timestamp  timestamp provided by DJI
+   * @return T_DjiReturnCode error code indicating if the subscription has been
+   * done correctly
+   */
+  T_DjiReturnCode avoid_data_callback(const uint8_t* data, uint16_t data_size,
+                                      const T_DjiDataTimestamp* timestamp);
+
+  /**
+   * @brief Retrieves the fused sea level altitude in meters.
+   * @param data pointer to T_DjiFcSubscriptionAvoidData data
```
[the remark this answers is not in the document]
Comment (biancabnd) [reply psdk_wrapper/include/psdk_wrapper/psdk_wrapper.hpp:996]: copy paste errors! nice catch

**Review by amoramar — APPROVED** [review]

**Pull request merged**